documentation/src/main/asciidoc/topics/con_estimating_memory_overhead.adoc

```diff
@@ -0,0 +1,18 @@
+[id='memory-overhead_{context}']
+= Memory overhead
+
+Memory overhead is additional memory that {brandname} uses to store entries.
+When you configure a lifespan for entries, for example, {brandname} adds timestamp metadata that increases memory overhead by a few bytes.
+Likewise {brandname} creates an additional linked list to track entries in the cache and perform eviction when you bound cache size.
+
+In general this memory overhead is not a significant factor for deployment planning and calculating the size of your data set.
+When you start measuring caches and tuning JVM memory configuration, this memory overhead comes into account.
+You cannot determine the amount of memory overhead that {brandname} adds to entries upfront because it depends on several factors.
+
+The only way to find any kind of exact amount of memory overhead involves JVM heap dump analysis.
```

Comment:
@wburns I wonder how feasible it would be to implement an API that takes a sample of say 1000 entries from the data container iterator and estimates the per-entry overhead for each of them, then returns the average?